admin/c2cgeoportal_admin/routes.py

```diff
@@ -3,7 +3,10 @@
 
 
 def includeme(config):
-    config.add_static_view('node_modules', 'c2cgeoportal_admin:node_modules')
+    config.add_static_view('node_modules_for_insider', 'c2cgeoportal_admin:node_modules')
+    config.add_static_view(
+        'node_modules_for_outsider',
+        '{}:node_modules'.format(config.root_package.__name__))
```

Comment:
Are you sure it make sense to add two static views with same name but different path.